### DISCUSSION TITLE
Add 'Open in Browser' option to Repos & Files

### DISCUSSION THREAD

**Discussion by michael-azogu** [community/community#33596] [Mobile] [answered]
The current UI forces me to copy url & paste in browser. Takes 10secs 🥲

**Comment by eliperkins** [comment, marked as answer]
Ooooh, this is great feedback @michael-azogu. Copy-and-pasting URLs to open in browser is definitely a bit of friction.

Can I get some more info from you to help get this fixed up? Can you share what operating system you're on (iOS vs Android + OS version), as well as what version of GitHub Mobile you're running (visible in the footer in Profile > Settings)? That'll help me make sure we address this fix where you're having this problem!

**Reply by michael-azogu** [reply]
Thanks!

it's Android 11 & GitHub v1.81.0-beta (10085)
